auth/signup.spec.js

```diff
@@ -0,0 +1,86 @@
+process.env.NODE_ENV = 'test'
+
+var chai = require('chai')
+var chaiHttp = require('chai-http')
+var server = require('../index')
+var cleandb = require('../utils/cleandb')
+var should = chai.should()
+
+chai.use(chaiHttp)
+
+describe('Auth signup tests', function() {
+
+  beforeEach(function() {
+    return cleandb()
+  })
+
+  it('should return error due the one empty field (birthday)', function(done) {
+    chai.request(server)
+    .post('/auth/signup')
+    .send({
+      name: 'Viking',
+      email: 'viking@garage.com',
+      password1: 'secret',
+      password2: 'secret',
+    })
+    .end(function(err, res) {
+      res.should.have.status(400)
+      res.should.be.json
+      res.body.error.should.equal(true)
```

Comment:
I think we can test if the message body actually contains a specific string, because some tests will have very identical handlers. We should differentiate them.